_posts/0000-01-02-raswinnair25.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title:  "Welcome to our second slide!"
+---
+Your text
```

Comment:
I have created a branch , created a file and made a commit, and opened a pull request.